src/main/java/io/boomerang/security/util/MultiReadHttpServletRequest.java

```diff
@@ -325,6 +316,11 @@ public int getServerPort() {
     return request.getServerPort();
   }
 
+  @Override
+  public BufferedReader getReader() throws IOException {
+    return request.getReader();
+  }
+
   @Override
```

Comment:
Can you run me through this change?

Reply:
@corcoja ?

Reply:
@AndreiOla, I'm not entirely sure. Looking at GIT history, this is part of [this](https://github.com/boomerang-io/flow.service.workflow/commit/4dbb6508cc21aaa92d370555ecc33705b6fb8e89) commit, which was pushed by yourself.
https://github.com/boomerang-io/flow.service.workflow/commit/4dbb6508cc21aaa92d370555ecc33705b6fb8e89

Reply:
This is a replacement for this code:
```java
  @Override
   public BufferedReader getReader() throws IOException {
     ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(this.buffer);
     return new BufferedReader(new InputStreamReader(byteArrayInputStream));
   }
```